src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Controllers/ManageController.cs

```diff
@@ -211,11 +232,6 @@ public async Task<IActionResult> SetPassword(SetPasswordViewModel model)
                 throw new ArgumentException($"Nie można załadować użytkownika z identyfikatorem '{this.accountService.GetUserId(this.User)}'.");
```

Comment:
please change it to ApplicationException

Reply:
Using "ApplicationExeption" brings error CA2201:Exception type System.ApplicationException is not sufficiently specific.

Reply:
ok, so use InvalidOperationException